GPL/vsconfig.gradle

```diff
@@ -32,7 +32,7 @@ def configureVisualStudio() {
 			println "Visual Studio not found!"
 			return
 		}
-		def vswhereOutput = "${vswherePath} -latest -format json".execute().text.trim()
+		def vswhereOutput = "${vswherePath} -products * -latest -requires Microsoft.VisualStudio.Component.VC.Tools.x86.x64 -format json".execute().text.trim()
 		def vswhereJson = new groovy.json.JsonSlurper().parseText(vswhereOutput);
 		if (vswhereJson.isEmpty()) {
 			println "Visual Studio not found!"
```

Comment:
This error message is a little confusing.  I installed the build tools with 0 of 3 required components, and got this error message.  VS was soft of installed...it just didn't have the MSVC.  I think it would be good if we directly tied the wording of this error message to the `-requires` argument somehow.